public/resources/remix/index.js

```diff
@@ -6,122 +6,121 @@
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var request = new XMLHttpRequest();
+    var url = metadata.host + "/projects/remix-bar";
+    var data = {
+      host: metadata.host,
+      id: metadata.projectId,
+      title: metadata.projectTitle,
+      author: metadata.projectAuthor,
+      updated: metadata.dateUpdated
+    };
+
+    request.onreadystatechange = function() {
+      if (request.readyState === XMLHttpRequest.DONE ) {
+        var response = request.response;
+        if (request.status !== 200) {
+          return callback({ status: request.status, response: response });
+        }
+        document.querySelector("body").insertAdjacentHTML("afterbegin", response);
+        callback(null);
       }
-    })
-    .done(function(response) {
-      $(response).prependTo("body");
-      callback(null, $);
-    })
-    .fail(function(jqXHR, textStatus) {
-      callback(textStatus);
-    });
+    };
+
+    var querystring = Object.keys(data).map(function(key) {
+      return encodeURIComponent(key) + "=" + encodeURIComponent(data[key]);
+    }).join("&");
+    url = url +  "?" +querystring;
+    request.open("GET", url);
+    request.send(data);
+  }
+
+  function injectStyleSheets(metadata) {
+    var stylesheet = document.createElement("link");
+    stylesheet.rel = "stylesheet";
+    stylesheet.href = metadata.host + "/resources/remix/clean-slate.min.css";
+    document.head.appendChild(stylesheet);
+
+    stylesheet = document.createElement("link");
+    stylesheet.rel = "stylesheet";
+    stylesheet.href = metadata.host + "/resources/remix/style.css";
+    document.head.appendChild(stylesheet);
   }
 
-  function injectStyleSheets($, metadata) {
-    var stylesheets =
-      "<link href=\"" + metadata.host + "/resources/remix/clean-slate.min.css\" rel=\"stylesheet\">\n" +
-      "<link href=\"" + metadata.host + "/resources/remix/style.css\" rel=\"stylesheet\">\n";
-    $("head").append(stylesheets);
+  function hasDataRemixAttribute(element) {
+    var name = element.getAttribute("name");
+    return /^data-remix-.+/.test(name);
   }
 
-  function getMetadata($) {
+  function getMetadata() {
     var metadata = {};
-    var metaTags = $("meta");
-
-    $.grep(metaTags, function(elem, index) {
-      return /^data-remix-.+/.test($(metaTags[index]).attr("name"));
-    })
-    .forEach(function(metaTag) {
-      metaTag = $(metaTag);
-      var key = metaTag.attr("name").replace(/^data-remix-/, "");
+    var metaElements = document.getElementsByTagName("meta");
+    var remixElements = Array.from(metaElements).filter(hasDataRemixAttribute);
```

Comment:
Change this to `Array.prototype.filter.call(metaElements, hasDataRemixAttribute);